### PR TITLE
feat(TG-265) save last value in backend fix

### PR DESCRIPTION
**Pull request by vreshch**
- save last value from output stream
- fix tests running in Windows

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/garage-it/SmartHouse-backend/pull/87" target="_blank">ReviewNinja

File: src/API/files/files.service.test.js [hunk 1]
```diff
@@ -32,7 +33,7 @@ describe('files service', () => {
         });
 
         it('should clean files folder content', () => {
-            del.should.calledWith(`${config.filesPath}/**`);
+            del.should.calledWith(path.join(config.filesPath,'/**'));
```
Comment (Appius) [inline src/API/files/files.service.test.js]: I think we should remove '/' from '/**'.
-- path.join(config.filesPath,'**')
!fix

Comment (Appius) [inline src/API/files/files.service.test.js]: !fixed

File: src/devices/saveDeviceLastValue.js [hunk 2]
```diff
@@ -1,10 +1,13 @@
...
 import Sensor from '../API/sensors/sensor.model';
 import input from '../data-streams/input';
+import output from '../data-streams/output';
 
 const DEVICE_STATUS_EVENT = 'status';
 
 export default function () {
-    input.stream
+
+    Rx.Observable.merge(input.stream, output.stream)
```
Comment (disist) [inline src/devices/saveDeviceLastValue.js]: Have you checked it?

It will be better to cover usage ```output.stream``` in test

Comment (vreshch) [inline src/devices/saveDeviceLastValue.js]: Sure I've check, the test were added as well

Comment (disist) [inline src/devices/saveDeviceLastValue.js]: Thanks

**Pull request merged**